### PR TITLE
Add documentation infos

### LINKED ISSUE #32: Update README for current code and deployment
See and copy relevant parts from https://github.com/hbz/lobid-organisations

### PR DESCRIPTION
**Pull request by SBRitter**
Add infos about  build, deploy, index, and eclipse import to the existing readme.

Will resolve #32

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,47 @@
...
 * Longitude
 * Post code
+
+The application is built using the Playframework and Elasticsearch.
+
+## Setup
+
+Create and change into a directory where you want to store the project, e.g.:
+
+``mkdir ~/git ; cd ~/git``
```
Comment (fsteeg) [inline README.md]: Are the double backticks on purpose? One is enough for `code`.

File: README.md [hunk 2]
```diff
@@ -1,10 +1,47 @@
...
+
+Get the project from GitHub:
+
+``git clone https://github.com/hbz/lobid-organisations.git``
```
Comment (fsteeg) [inline README.md]: Wrong repo here.

**Comment by SBRitter** [comment]
Revised the readme file.

**Comment by fsteeg** [comment]
+1

**Pull request merged**